diff --git a/src/makefile.mingw b/src/makefile.mingw
index 942e138ef..5a212ec31 100644
--- a/src/makefile.mingw
+++ b/src/makefile.mingw
@@ -203,6 +203,7 @@ LIBS += -lregex
 # Code may not compile with -D_FORTIFY_SOURCE=2
 #CFLAGS += -D_FORTIFY_SOURCE=2
 #LUACFLAGS += -D_FORTIFY_SOURCE=2
+LDFLAGS += -lssp

 # Added the according preprocessor for 'preproc' - C. Blue
 # If your system doesnt have a dedicated cpp command, you might
