diff --git a/src/common/defines-local.h b/src/common/defines-local.h
index 8e1bbd4fe..1e9a0ba34 100644
--- a/src/common/defines-local.h
+++ b/src/common/defines-local.h
@@ -25,6 +25,6 @@


 /* Permanent event "Ironman Deep Dive Challenge": */
-#define WPOS_IRONDEEPDIVE_X	63
-#define WPOS_IRONDEEPDIVE_Y	63
+#define WPOS_IRONDEEPDIVE_X	30
+#define WPOS_IRONDEEPDIVE_Y	34
 #define WPOS_IRONDEEPDIVE_Z	-1
